File: programacao/c/respostas/selecao/42.c
#include <stdio.h>

int main(void) {
    float albumina, creatinina, rac;

    printf("Albumina (mg): ");
    scanf("%f", &albumina);
    printf("Creatinina (g): ");
    scanf("%f", &creatinina);

    rac = albumina / creatinina;

    printf("RAC: %.1f\n", rac);

    if (rac < 30.0) {
        printf("Normal\n");
    } else if (rac < 300.0) {
        printf("Microalbuminuria\n");
    } else {
        printf("Macroalbuminuria\n");
    }

    return 0;
}

File: programacao/python/respostas/selecao/42.py
albumina = float(input("Albumina (mg): "))
creatinina = float(input("Creatinina (g): "))

rac = albumina / creatinina

print(f"RAC: {rac:.1f}")

if rac < 30.0:
    print("Normal")
elif rac < 300.0:
    print("Microalbuminuria")
else:
    print("Macroalbuminuria")
